src/decorators.ts

```diff
@@ -1,14 +1,14 @@
 import "reflect-metadata";
 
 import * as crypto from "crypto";
```

Comment:
Are there any functional changes to this file?  I see a lot of whitespace changes, but can't tell if there are also real changes.

Reply:
Yes, I had to add change the parameter type from Function[] to Function[] | {[key: string]: object} to accommodate Drizzle tables, since they aren't class-based.

Reply:
Got it!  Can you revert the whitespace changes in all files and leave only the functional changes?  I know it's annoying, but they make it hard for us to understand the PR and what changes were actually made.